packages/notifi-dapp-example/src/components/SubscriptionValueInput.tsx

```diff
@@ -34,26 +52,112 @@ export const SubscriptionValueInput: React.FC<SubscriptionValueInputProps> = (
       >
         {props.subscriptionValue
           ? props.subscriptionValue.label
-          : <div className="text-notifi-text-light">Select Pair</div> ?? (
-              <div className="text-notifi-text-light">Select Pair</div>
+          : (
+              <div className="text-notifi-text-light">
+                {getSubscriptionValueOrRef() === 'pricePairs'
+                  ? 'Select Pair'
+                  : 'Select Market'}
+              </div>
+            ) ?? (
+              <div className="text-notifi-text-light">
+                {getSubscriptionValueOrRef() === 'pricePairs'
+                  ? 'Select Pair'
+                  : 'Select Market'}
+              </div>
```

Comment:
Looks like these two statements are the same. Do I miss anything? 
And just my opinion, using such kind of condition rendering for label empty state is not scalable.
Do you think we can just extract the value from the ref? like
```ts
function splitCamelCaseToString(input) {
  return input
    .replace(/([a-z])([A-Z])/g, '$1 $2')
    .toLowerCase();
}
 
 <div>Select {splitCamelCaseToString(getSubscriptionValueOrRef())} </div>

Reply:
Ah you are right, I need to remove one of the statement
The label is customized for different topics right now, so we can not extract the value from ref, I think in the future we may need to include a new field in metadata to include the select input placeholder.

Reply:
Yeah, I understand the label is customized. But what I am thinking is that if they set the ref in AP is `pricePair`, the empty label showing on the UI would be `Select Price Pair`. so it will separate the camel case to a normal phase. 
So it can be a default behavior which is controllable through AP using the `input ref`

Reply:
Ah you mean we are doing something like this?

```
function splitCamelCaseToString(input) {
  return input
    .replace(/([a-z])([A-Z])/g, '$1 $2')
    .toLowerCase();
}
( <div className="text-notifi-text-light">
      {getSubscriptionValueOrRef() === 'pricePairs'
             ? 'Select Pair'
             : 'Select Market'}
 </div>
) ?? (
  <div>Select {splitCamelCaseToString(getSubscriptionValueOrRef())} </div>)
```